package.json

```diff
@@ -170,7 +170,7 @@
         "lit-html": "^2.4.0",
         "mocha-junit-reporter": "^2.0.2",
         "netlify-cli": "^15.9.0",
-        "next": "^14.0.4",
+        "next": "^14",
```

Comment:
Oh, actually, this interacts negatively with the lock, still. 🙈

Do you want this to specifically be `^14`? In which case we need a `yarn install` to recreate the lock from here, or we could turn it back to `^14.0.4` and then updates in the dependabot branch will hopefully cover us here. _I think._

Reply:
Ah, sorry. I forgot to check in my updated yarn.lock file.